virtool/subtractions/api.py

```diff
@@ -35,143 +41,197 @@
 BASE_QUERY = {"deleted": False}
 
 
-@routes.get("/subtractions")
-async def find(req):
-    db = req.app["db"]
+@routes.view("/subtractions")
+class Subtractions(PydanticView):
+    async def get(self) -> r200[VirtoolSubtractionMinimal]:
```

Comment:
The `Virtool` prefix models are intended for use within Virtool, not for responses. The idea is the responses can be used to populate the models in other libraries that import `virtool-core`, like `virtool-workflow`.
```suggestion
    async def get(self) -> r200[SubtractionMinimal]:
```